server/v2/stf/stf_router.go

```diff
@@ -36,9 +36,11 @@ func NewMsgRouterBuilder() *MsgRouterBuilder {
 }
 
 type MsgRouterBuilder struct {
-	handlers     map[string]MsgHandler
-	preHandlers  map[string][]PreMsgHandler
-	postHandlers map[string][]PostMsgHandler
+	handlers           map[string]MsgHandler
+	globalPreHandlers  []PreMsgHandler
+	preHandlers        map[string][]PreMsgHandler // TODO document how to do ordering, if needed
+	postHandlers       map[string][]PostMsgHandler
+	globalPostHandlers []PostMsgHandler
```

Comment:
why are global post handlers here? this would mean that antehandlers need to exist here as well no?